### PR TITLE
add compression to express and minification of html

### LINKED ISSUE #73: minimize html

### PR DESCRIPTION
**Pull request by butlerx**
close #73 and fix html errors

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -64,18 +64,46 @@ feed:
...
   enable: true
   exclude:
     - '*.min.css'
-neat_js:
+nfc_js:
   enable: true
   mangle: true
   exclude:
     - '*.min.js'
+hfc_favicons:
```
Comment (SeanHealy33) [inline _config.yml]: what does all this do out of interest?

Comment (butlerx) [inline _config.yml]: generate favicons

File: _config.yml [hunk 2]
```diff
@@ -64,18 +64,46 @@ feed:
...
-  enable: false
-neat_css:
+nfc_html:
+  enable: true
+nefc_css:
   enable: true
   exclude:
     - '*.min.css'
-neat_js:
+nfc_js:
```
Comment (SeanHealy33) [inline _config.yml]: should this be `hfc`?

File: _config.yml (hunk 1 above)
Comment (SeanHealy33) [inline _config.yml]: ahh cool makes sense

File: _config.yml (hunk 2 above)
Comment (butlerx) [inline _config.yml]: yes it should

**Pull request merged**